src/bankAccount.test.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const BankAccount = require('./bankAccount');
+
+test('Should return account info', () => {
+  const account = new BankAccount('Dmytro', '26.11.1994', 1000);
+
+  expect(account.getInfo())
+    .toBe(`Name: Dmytro, Age: 26, Amount: 1000$`);
```

Comment:
move Dmytro and 1000 to variables and use here or do not use template string

Reply:
@YuriiHoliuk It is a test to it is ok to use hardcoded values

Reply:
But the test depends on current date 
So it will be broken at some moment